### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rudolfs**
✨

### CODE REVIEW

**Comment by rudolfs** [comment]
The visual changes weren't expected. Might be either a glitch in the build or maybe something changed in the browser rendering. In any case, doesn't seem like a big deal, it's just some aliasing on the settings icon.

Screenshot 2023-05-30 at 10 34 09

**Review by sebastinez — APPROVED** [review]

**Comment by sebastinez** [comment]
> The visual changes weren't expected. Might be either a glitch in the build or maybe something changed in the browser rendering. In any case, doesn't seem like a big deal, it's just some aliasing on the settings icon.
> 
> Screenshot 2023-05-30 at 10 34 09

It seems to me this has to do with some of the changes to `happy-dom` from what I saw on the CHANGELOG, or maybe also changes to browser.s

File: package.json [hunk 1]
```diff
@@ -19,33 +19,33 @@
     "node": ">=18.16.0"
   },
   "devDependencies": {
-    "@playwright/test": "^1.33.0",
-    "@sinonjs/fake-timers": "^10.1.0",
-    "@sveltejs/vite-plugin-svelte": "^2.2.0",
+    "@playwright/test": "^1.34.3",
+    "@sinonjs/fake-timers": "^10.2.0",
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply package.json:23]: Hm, yeah, it seems to be related to us using [`@sinonjs/fake-timers`](https://github.com/sinonjs/fake-timers) in the tests to mock timers. Vite recommends asking the lib owners to fix it. But it's not clear to me how this should be fixed tbh.

>  If the module is imported from a third-party library (that's meant to be used in the browser), it's advised to report the issue to the respective library.

https://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility

[the remark this answers is not in the document]
Comment (rudolfs) [reply package.json:23]: Skipping the log messages in the logger for now: https://github.com/radicle-dev/radicle-interface/pull/795/files#diff-851c4beda1acd44bc9a1735d3130370435028df16cb3bd4c743095410bfad8f8R45.

**Pull request merged**